infrastructure/data/index.ts

```diff
@@ -10,6 +10,20 @@ const DB_ROOT_USERNAME = "dbuser";
 const env = pulumi.getStack();
 const networking = new pulumi.StackReference(`planx/networking/${env}`);
 
+// ShareDB does not use SSL - from Postgres 16 onwards this is forced on my default
+// Create a parameter group which turns this setting off
+const parameterGroup = new aws.rds.ParameterGroup("parameterGroup", {
+  name: "postgres16-force-ssl-off",
+  family: "postgres16",
```

Comment:
My only thought here is whether the 'default' parameter group prescribed by Pulumi may or may not have any differences from `postgres16`? Seems unlikely. I also wonder if there's any merit to doing this slightly differently, as per the example [here](https://www.pulumi.com/registry/packages/aws/api-docs/rds/instance/#rds-db2-usage)) (again, I think probably unnecessary, just thinking out loud).

Out of curiosity, you find the family `postgres16` by running the following command as per [here](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbparametergroup.html#aws-resource-rds-dbparametergroup-properties)?

```
aws rds describe-db-engine-versions --query "DBEngineVersions[].DBParameterGroupFamily" --engine postgres
```